lib/utils.pm

```diff
@@ -2371,16 +2372,29 @@ This functions checks if ca-certificates-suse is installed and if it is not it a
 
 sub ensure_ca_certificates_suse_installed {
     return unless is_sle || is_sle_micro;
+    my $ret = 0;
     if (script_run('rpm -qi ca-certificates-suse') == 1) {
         my $host_version = get_var("HOST_VERSION") ? 'HOST_VERSION' : 'VERSION';
         my $distversion = get_required_var($host_version) =~ s/-SP/_SP/r;    # 15 -> 15, 15-SP1 -> 15_SP1
-        zypper_call("ar --refresh http://download.suse.de/ibs/SUSE:/CA/SLE_$distversion/SUSE:CA.repo");
-        if (is_sle_micro) {
-            transactional::trup_call('--continue pkg install ca-certificates-suse');
+                                                                             # exit codes defined will not stop on error.
+        my $res = zypper_call("ar --refresh http://download.suse.de/ibs/SUSE:/CA/SLE_$distversion/SUSE:CA.repo", retry => 3, exitcode => [0, 1, 2, 3, 4]);
```

Comment:
Using both $res and $ret hinders readability, in my opinion. Since $res is only used once, what do you think about moving the zypper_call into the if statement?

Reply:
the zypper exit code is the basilar value for block triggering and moving that command into the condition is much more readability hindering.
I agree that variables names are too similar, I will rename `res`.

Reply:
I renamed a variable.
Also, let's image that long zypper command into the `if` matching 0: on that right side of the screen we will lost :slightly_smiling_face: